test/post.js

```diff
@@ -6,7 +6,13 @@ var g;
 var p;
 
 before(function(done) {
-  when.then(function(data) { g = data.data; p = g.posts[0]; done(); },
+    when.then(function(data) {
+	g = data.data;
+	p = g.posts[0];
+	post_missing_both_post_fields = g.posts[1];
```

Comment:
True.. I wonder if we should add fallback to that too. But that's for another PR. `post_missing_post_date_fields`?